hathor/p2p/sync_version.py

```diff
@@ -47,6 +47,14 @@ def get_priority(self) -> int:
         else:
             raise ValueError('value is either invalid for this enum or not implemented')
 
+    def is_v1(self) -> bool:
+        """Return True for V1 and V1_1."""
+        if self is SyncVersion.V1:
+            return True
+        if self is SyncVersion.V1_1:
+            return True
+        return False
```

Comment:
Maybe it's a non-issue, but if for any reason we need to release something like a sync-v1.2 before sync-v2 is out, we would need to remember to update this code. Maybe negating the condition would be safer, something like `if not sync-v2`.

Reply:
Good point. I just changed it to use `get_priority()`. What do you think?

Reply:
Great, that priority method is perfect for this. Thanks!